packages/omniapi/src/public/Datasets_Branch.ts

```diff
@@ -26,7 +26,7 @@ const _listBranches: $OmniMethod<
     datasetRid: $C.DatasetRid,
     $queryParams?: { pageSize?: $C.PageSize; pageToken?: $C.PageToken },
   ) => Promise<$C.ListBranchesResponse>
-> = [0, "/v1/datasets/{0}/branches"];
+> = [0, "/v1/datasets/{0}/branches", 2];
```

Comment:
I'm confused as to why these numbers have been appended to the end in these generated files?

Reply:
They tell the runtime if there is query or header params. Its just bitwise math.